experiments/ClimaEarth/setup_run.jl

```diff
@@ -496,46 +496,33 @@ function CoupledSimulation(config_dict::AbstractDict)
         The concrete steps for proper initialization are:
         =#
 
-        # 1.coupler updates surface model area fractions
+        # 1. Coupler updates surface model area fractions
         FieldExchanger.update_surface_fractions!(cs)
 
-        # 2.surface density (`ρ_sfc`): calculated by the coupler by adiabatically extrapolating atmospheric thermal state to the surface.
-        # For this, we need to import surface and atmospheric fields. The model sims are then updated with the new surface density.
+        # 2. Import atmospheric and surface fields into the coupler fields, then broadcast them back out to all components.
         FieldExchanger.import_combined_surface_fields!(cs.fields, cs.model_sims)
         FieldExchanger.import_atmos_fields!(cs.fields, cs.model_sims)
         FieldExchanger.update_model_sims!(cs.model_sims, cs.fields)
 
-        # Set all initial cache values for the land model, now that we have updated drivers
+        # 3. Set all initial cache values for the land model, now that we have updated drivers
         land_set_initial_cache! = CL.make_set_initial_cache(cs.model_sims.land_sim.model)
         land_set_initial_cache!(
             cs.model_sims.land_sim.integrator.p,
             cs.model_sims.land_sim.integrator.u,
             cs.model_sims.land_sim.integrator.t,
         )
 
-        # 3.surface vapor specific humidity (`q_sfc`): step surface models with the new surface density to calculate their respective `q_sfc` internally
-        ## TODO: the q_sfc calculation follows the design of the bucket q_sfc, but it would be neater to abstract this from step! (#331)
-        Interfacer.step!(land_sim, tspan[1] + Δt_cpl)
-        Interfacer.step!(ocean_sim, tspan[1] + Δt_cpl)
-        Interfacer.step!(ice_sim, tspan[1] + Δt_cpl)
-
-        # 4.turbulent fluxes: now we have all information needed for calculating the initial
-        # turbulent surface fluxes
+        # 4. Compute radiative fluxes and update the coupler fields and model simulations with the new fluxes
```

Comment:
Maybe let's mention that one should run all the other callbacks that change p as well